plaso/parsers/gdrive_synclog.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+"""Parser for Google Drive Sync log files."""
+
+from __future__ import unicode_literals
+
+import logging
+
+import pyparsing
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import errors
+from plaso.lib import definitions
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+class GDriveSyncLogEventData(events.EventData):
+  """GDrive Sync log event data.
+
+  Attributes:
+    time (str): time of event (string, time with offset and timezone of user).
+    log_level (str): logging level of event (e.g., DEBUG, WARN, INFO, ERR).
+    pid (int): process ID of process which logged event.
+    thread (str): ID:name of thread which logged event (colon-separated).
+    source_code (str): filename:line_number of source file which logged event.
```

Comment:
maybe source_code_reference ?

Reply:
I left this as source_code because it's how SkyDrive's parser did it and had the same convention in the log, so I felt like keeping the field name the same would be more consistent. LMK if you feel strongly and I can change it.

Ref: https://github.com/log2timeline/plaso/blob/master/plaso/parsers/skydrivelog.py#L27